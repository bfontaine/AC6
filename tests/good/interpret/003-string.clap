val s = "This should be easy, right?"
