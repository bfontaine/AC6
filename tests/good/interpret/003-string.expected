:- s = "This should be easy, right?"
